### PR TITLE
Remove cache-clearing-service from Carrenza backends

### PR DESCRIPTION
**Pull request by schmie**
- This service was migrated to AWS as it requires direct access to the
migrated Cache instances.

solo: @schmie

### CODE REVIEW

File: hieradata/common.yaml [hunk 1]
```diff
@@ -18,7 +18,6 @@ node_class: &node_class
   backend:
     apps:
       - asset-manager
-      - cache-clearing-service
```
Comment (cbaines) [inline hieradata/common.yaml]: I don't think removing the application here will get Puppet to remove it from the machine, you'll need to leave this here, but set the `enabled` parameter to false, then deploy that change, then remove it from the list here.

Comment (fredericfran-gds) [reply hieradata/common.yaml]: govuk::apps::cache_clearing_service::enabled: false

Comment (fredericfran-gds) [reply hieradata/common.yaml]: I guess it is more clean this way

Comment (schmie) [reply hieradata/common.yaml]: Right, last time we did this, we worked with instances which were redeployed after - thanks for catching it!

Comment (cbaines) [reply hieradata/common.yaml]: The changes here https://github.com/alphagov/govuk-puppet/pull/9105 have now been deployed, so this is no longer an issue.

**Pull request merged**